sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -88,6 +102,16 @@ internal PageResult() { }
         public System.Collections.Generic.IReadOnlyList<T> Values { get { throw null; } }
         public static System.ClientModel.PageResult<T> Create(System.Collections.Generic.IReadOnlyList<T> values, System.ClientModel.ContinuationToken pageToken, System.ClientModel.ContinuationToken? nextPageToken, System.ClientModel.Primitives.PipelineResponse response) { throw null; }
     }
+    public abstract partial class RefreshableCredential : System.ClientModel.Credential
```

Comment:
could we combine the two abstractions (refreshable and not)? I am not sure it's worth having two types for such obscure API.

Reply:
We could. I expermented with separating them since not all credential types, across clouds especially, have a finite expiration.